CESC-submission/paper.tex

```diff
@@ -208,64 +208,61 @@ \section{Impact of Financial Incentives on Service Delivery}
+service, in the sense that the payment will be partially computed based on the amount of
 service provided. This requires Ursula to confirm its activity by proving that she served 
 a given number of distinct re-encryption requests during a given period. To make the distinction 
 clear, we refer to the first as \emph{confirm availability}, and for the second as \emph{confirm 
 service activity}. \\
 
 
-\noindent{\bf Confirm availability and service activity solutions.} In~\cite{confirm-activity-draft}, we introduce several solutions for both forms of the above problem. These solutions differ in the trust, efficiency, interactivity, and resource requirements, in addition to their security guarantees, which we present briefly in what follows. Before that, we review the adopted threat model.
+\noindent{\bf Confirm availability and service activity solutions.} In~\cite{confirm-activity-draft}, we introduce several solutions for both forms of the above problem. These solutions differ with respect to trust, efficiency, interactivity, and resource requirements, in addition to their security guarantees, which we present briefly in what follows. Before that, we review the adopted threat model.
 
 
 \paragraph{Threat model.} We do not place trust in any party and we assume that all participants are rational and self-interested. Thus, they may follow the protocol or deviate from it, either acting on their own or in collusion with each other, based solely on what maximizes their utility gain. 
 
 
-However, we argue that collusion between Bob and Ursula to perform accounting attacks (i.e., Bob pretends that service has been delivered allowing Ursula to collect payments for free) is not economically efficient, and hence, not an issue in our system. The reason is that the amount of work an Ursula does when replying to a re-encryption request is minimal. This means that colluding with Bob will save minimal resources, which makes this collusion case unappealing to rational Ursulas. This case is a great example of how traditional threat modeling and risk management practices differ from those economic-oriented ones in ruling out financially-inefficient threats.
+However, we argue that collusion between Bob and Ursula to perform accounting attacks (i.e., Bob pretends that service has been delivered allowing Ursula to collect payments for free) is not economically advantageous, and hence, not an issue in our system. The reason is that the amount of work an Ursula does when replying to a re-encryption request is minimal. This means that colluding with Bob to fake re-encryptions will save minimal resources and has no clear economic gain (such as extra subsidies), which makes this collusion case unappealing to rational Ursulas. This case is a great example of how traditional threat modeling and risk management practices differ from those economic-oriented ones in ruling out financially-inefficient threats.
```

Comment:
add: 
to fake re-encryptions will save minimal resources and has no clear economic gain (such as extra subsidies),